src/2_storage.js

```diff
@@ -136,39 +136,41 @@ BranchStorage.prototype['session'] = function() {
 	return webStorage(false);
 };
 
-var cookies = function(perm) {
+var cookies = function() {
 	var setCookie = function(key, value) {
-		var expires = '';
-		if (perm) {
-			var date = new Date();
-			date.setTime(date.getTime() + COOKIE_MS);
-			expires = '; branch_expiration_date=' + date.toGMTString() +
-				'; expires=' + date.toGMTString();
+		document.cookie = key + '=' + value + '; path=/';
+	};
+	var removeCookie = function(key, ignorePrefix) {
+		var expires = 'Thu, 01 Jan 1970 00:00:01 GMT';
+		if (ignorePrefix) {
+			document.cookie = key + '=; expires=' + expires + '; path=/';
+		}
+		else {
+			document.cookie = prefix(key) + '=; expires=' + expires + '; path=/';
 		}
-		document.cookie = key + '=' + value + expires + '; path=/';
 	};
 	return {
 		getAll: function() {
-			var cookieArray = document.cookie.split(';');
 			var returnCookieObject = { };
+			var cookieArray = document.cookie.split(';');
 			for (var i = 0; i < cookieArray.length; i++) {
-				var cookie = cookieArray[i].replace(' ', '');
-				cookie = cookie.substring(0, cookie.length);
-				if (cookie.indexOf(BRANCH_KEY_PREFIX) !== -1) {
```

Comment:
any reason we got rid of this `BRANCH_KEY_PREFIX` check?

Reply:
I've included this check over at:

https://github.com/BranchMetrics/web-branch-deep-linking/blob/9fb097ae05626a8bfe79668d2f3fdc66de54def2/src/2_storage.js#L61

There is now a separate function which detects whether a cookie is a Branch cookie. We use it at the following locations:

https://github.com/BranchMetrics/web-branch-deep-linking/blob/9fb097ae05626a8bfe79668d2f3fdc66de54def2/src/2_storage.js#L173

https://github.com/BranchMetrics/web-branch-deep-linking/blob/9fb097ae05626a8bfe79668d2f3fdc66de54def2/src/2_storage.js#L200